samples/Templates/Scenarios/CalendarReminder.template.json

```diff
@@ -1,38 +1,31 @@
 {
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
-	"version": "1.0",
+	"version": "1.3",
 	"speak": "<s>Your  meeting about \"Adaptive Card design session\"<break strength='weak'/> is starting at ${formatDateTime(start.dateTime, 'HH:mm')}pm</s><s>Do you want to snooze <break strength='weak'/> or do you want to send a late notification to the attendees?</s>",
 	"body": [
 		{
 			"type": "TextBlock",
 			"text": "${summary}",
 			"size": "Large",
 			"weight": "Bolder",
-			"wrap": true,
```

Comment:
for accessibility reasons having wrap in this element may be required

Reply:
Sounds good, I'll make sure wrap is included on the TextBlocks